Makefile

```diff
@@ -31,7 +31,6 @@ export VALIDATE_ORIGIN_BRANCH
 # make DOCKER_LDFLAGS="-X github.com/docker/docker/daemon/graphdriver.priority=overlay2,devicemapper" dynbinary
 #
 DOCKER_ENVS := \
-	-e BUILD_APT_MIRROR \
```

Comment:
Is this still needed for users that use this option? (or is it only used during `build`, in which case it needs to be added to the list of `--build-arg` (but without a value, so that it can inherit env variables with that name)

Reply:
I don't see it being used anywhere in our code base. I think it was just used to build debian packages which has been moved to docker-ce-packaging repo: https://github.com/moby/moby/pull/26375/files#diff-01ca5adb236edbbea85ba055aa48330de763cab402a67403855aa3a887ed6a07R58